### PR TITLE
90% SHIP-140: Talis Customisations

### PR DESCRIPTION
**Pull request by danielmatthew**
A Sass-based, Talis-flavoured version of Bootstrap 3 for use in projects. All customisations are baked in, so only app-specific components should be required.

## Todo
- [x] Alerts
- [x] Buttons
- [x] Modals
- [x] Forms
- [x] Type
- [x] Labels
- [x] Tables
- [x] Navs
- [x] Dropdowns
- [x] Pagination
- [x] Navbar
- [x] Tidy navbar mobile styles - might as well get that conformant.
- [x]  Pass @jeremybaines UX review

While a picture speaks a thousand words, you can also visit the contents of the `gh-pages` branch: https://talis.github.io/bootstrap-sass/

screencapture-talis-github-io-bootstrap-sass-1465577890442

### CODE REVIEW

**Comment by danielmatthew** [comment]
https://talis.github.io/bootstrap-sass/

File: assets/stylesheets/bootstrap/_variables.scss [hunk 1]
```diff
@@ -574,13 +584,13 @@ $label-default-bg:            $gray-light !default;
 //** Primary label background color
 $label-primary-bg:            $brand-primary !default;
 //** Success label background color
-$label-success-bg:            $brand-success !default;
+$label-success-bg:            #a0cfbb !default;
```
Comment (danielmatthew) [inline assets/stylesheets/bootstrap/_variables.scss]: These three colours aren't the product of a consistent transformation on variables like `$brand-success`

File: assets/stylesheets/bootstrap/mixins/_alerts.scss [hunk 2]
```diff
@@ -1,10 +1,8 @@
 // Alerts
 
-@mixin alert-variant($background, $border, $text-color) {
+@mixin alert-variant($background, $border) {
```
Comment (danielmatthew) [inline assets/stylesheets/bootstrap/mixins/_alerts.scss]: Dropped `$text-color` as all alerts are set in the body text colour.

File: assets/stylesheets/bootstrap/mixins/_buttons.scss [hunk 3]
```diff
@@ -3,37 +3,73 @@
 // Easily pump out default styles, as well as :hover, :focus, :active,
 // and disabled options for all buttons
 
-@mixin button-variant($color, $background, $border) {
+@mixin button-variant-plain($color, $background, $border) {
```
Comment (danielmatthew) [inline assets/stylesheets/bootstrap/mixins/_buttons.scss]: `btn-default` and `btn-danger` taken on a different style to the others.

**Comment by junglebarry** [comment]
I have a couple of small requests:

* The pills and pagination all redirect to `#`, which forces them to the top of the page.  Is it worth having them all address their own section, so a click on them doesn't cause a massive jump?
* Some extra padding at the bottom of the page would make the modal button clearer.

After that, I'd say @jeremybaines' feedback is much more important than mine!

**Comment by junglebarry** [comment]
👍

**Pull request merged**